app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -23,14 +23,14 @@ sealed class Event {
         object SetAsDefault : GrowthData("xgpcgt")
 
         /**
-         * Event recording the first time Firefox has been resumed in a 24 hour period.
+         * Event recording that a URI has been loaded.
          */
-        object FirstAppOpenForDay : GrowthData("41hl22")
+        object UriLoaded : GrowthData("")
 
         /**
-         * Event recording the first time a URI is loaded in Firefox in a 24 hour period.
+         * Event recording that an ad was clicked in a search engine results page.
          */
-        object FirstUriLoadForDay : GrowthData("ja86ek")
+        object SerpAdClicked : GrowthData("")
```

Comment:
Out of interest, how is the token name generated, and do these require token names adding?

Reply:
The token name is generated in the Adjust web app, usually by the marketing/data team. I will wait to land this until those strings have been delivered. Without the tokens, I assume sending the events to the Adjust backend will be no-ops

Reply:
Gotcha, thanks 👍